lca-chimevc-stack/lambda_functions/chime_call_analytics_initialization/index.js

```diff
@@ -599,7 +599,10 @@ const handler = async function handler(event, context) {
       callData = await getCallDataFromChimeEventsWithLambdaHook(event);
 
       console.log('Saving callData to DynamoDB');
-      await writeCallDataToDdb(callData);
+      await writeCallDataToDdb(callData, callData.callId);
+      if(callData.callId !== callData.originalCallId) {
+        await writeCallDataToDdb(callData, callData.originalCallId);
```

Comment:
Would it be better if we always write a mapping entry here?
HashKey is always the VC callId, originalCallId. This entry will be used in End event below to translate from VC callId to customer modified callId.
The callId field is the callData.callId. Thus, the callId field value is either the VC callId if the customer did not modify the callId, or the modified callId.

Reply:
Then in the Event message. The code will always look up the mapping entry using the VC callId to get the callId that should be sent to the KDS event.